tests/test_helpers.py

```diff
@@ -38,3 +60,144 @@ def test_uppercases():
 
     bad_letters = "#$!^%#tables"
     assert set(bad_letters).difference(uppercases)
+
+
+def test_get_font_stack_raises(font_stack_names):
```

Comment:
It looks like `font_stack_names` might not be used here

Reply:
Good spot 😅.